herd/tests/instructions/AArch64/A162.litmus

```diff
@@ -0,0 +1,11 @@
+AArch64 A162
+(* Test load exclusive; Store exclusive, symbolic location *)
+
+ { 0:X4=z; uint64_t z=2; 1:X5=5; 1:X6=z}
```

Comment:
The initialisation `1:X5=5` implicitly declares register `1:X5` to be of type `int`. As register is used int its `X` form by`P1`, the overzealous type checker of litmus detects   a contradiction and issues a warning.

The situation is in fact harmless. Better silence the warning anyway:
```suggestion
 { 0:X4=z; uint64_t z=2; int64_t 1:X5=5; 1:X6=z}
```